### PR TITLE
Bug 1223955 - Wait until Airplane mode menu becomes active in Settings app

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=652a188b1032624efa9c7bebbabd374022d14d68

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py [hunk 1]
```diff
@@ -85,9 +85,8 @@ def launch(self):
         Wait(self.marionette).until(
             expected.element_present(*self._app_loaded_locator))
 
-        # this is located at the end of the page.  If this is shown, everything is rendered.
-        Wait(self.marionette).until(
-            expected.element_present(*self._bluetooth_menu_item_locator))
+        # Wait for airplane mode menu-item to become enabled
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py]: This comment doesn't add value to the line below. However I think having a comment is interesting: why do we wait for this particular element to be ready? Is it the longest to be ready?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py]: Something to do with imagecompare, from what I understood from No-Jun. I'll wait on his comment then.

**Comment by npark-mozilla** [comment]
We need this PR so when running Imagecompare tests, it can consistently capture the screenshot where the switch is active (and disabled)

**Comment by npark-mozilla** [comment]
Also, it is the longest one to be ready.

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py (hunk 1 above)
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py]: Based on @npark-mozilla 's comment, an explanation like: 
`# Airplane mode is the latest item to be ready.`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0a754e44444b06a38bdf62565cdde00920543b54

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7b80246ac0a958782d4444ef55e170674db782b0

**Pull request merged**